cpp/ql/lib/experimental/semmle/code/cpp/semantic/analysis/RangeAnalysisStage.qll

```diff
@@ -73,6 +73,7 @@ import experimental.semmle.code.cpp.semantic.SemanticCFG
 import experimental.semmle.code.cpp.semantic.SemanticType
 import experimental.semmle.code.cpp.semantic.SemanticOpcode
 private import ConstantAnalysis
+private import semmle.code.cpp.Location
```

Comment:
This should probably also be a language-neutral location. The other `cpp`-related imports are just to compensate for the fact that we can't import something from `../` in QL, so this is also the only language-specific import in the file.